.pre-commit-config.yaml

```diff
@@ -20,3 +20,38 @@ repos:
   rev: v2.17.2
   hooks:
   - id: pylint
+- repo: local
```

Comment:
Can we add docs on this in developer docs as well as clear instructions on how to opt out?

Reply:
Added a note on skipping pre-commit workflow as and when needed. :)